src/engine/RuntimeInformation.h

```diff
@@ -53,7 +53,8 @@ class RuntimeInformation {
   void writeToStream(std::ostream& out, size_t indent = 1) const {
     using json = nlohmann::json;
     out << indentStr(indent) << '\n';
-    out << indentStr(indent - 1) << u8"├─ " << _descriptor << '\n';
+    out << indentStr(indent - 1) << reinterpret_cast<const char*>(u8"├─ ")
+        << _descriptor << '\n';
```

Comment:
Why do you keep the u8 prefix here, but not in, say, `Tokenizer.h` ?

Reply:
No good reason, I will remove them.